piqe_ocp_lib/api/tasks/operator_ops.py

```diff
@@ -117,6 +117,32 @@ def is_operator_installed(self, operator_name: str, operator_namespace: str) ->
         """
         return self.csv.get_cluster_service_version(operator_name, operator_namespace) is not None
 
+    def get_version_of_operator(self, operator_name: str, operator_namespace: str) -> str:
+        """
+        Get the version of operator if operator is installed
+        :param operator_name: name of the operator.
+        :param operator_namespace: namespace of the operator
+        return: version of the operator
+        """
+        if self.is_operator_installed(operator_name, operator_namespace) is True:
```

Comment:
```suggestion
        if self.is_operator_installed(operator_name, operator_namespace):
```

this can be simplified